### DISCUSSION TITLE
Not able to take unicode characters or characters other than english language

### DISCUSSION THREAD

**Discussion by ratannahn1** [NotCookey/Quote2Image#4] [Ideas]
![hello](https://user-images.githubusercontent.com/118421208/211237087-8e32129d-66e3-43d8-8320-57255bb381c9.png)
PFA the image for sample. 

Can it include unicode support?

**Comment by NotCookey** [comment]
> ![hello](https://user-images.githubusercontent.com/118421208/211237087-8e32129d-66e3-43d8-8320-57255bb381c9.png) PFA the image for sample.
> 
> Can it include unicode support?

I will investigate the issue you brought to my attention regarding the rendering of Unicode characters. I have not previously tested for this, but I will see if theres a solution to handle Unicode characters in the new update.
